.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 80,
+  "semi": true,
+  "singleQuote": false,
+  "jsxSingleQuote": true,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "arrowParens": "avoid"
+}
```

Comment:
Please delete this file. Because Prettier is "opinioned" code formatter. We shouldn't customize it.